packages/api/src/routes/medical/internal-cw.ts

```diff
@@ -0,0 +1,125 @@
+import NotFoundError from "@metriport/core/util/error/not-found";
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import httpStatus from "http-status";
+import {
+  verifyCxProviderAccess,
+  verifyCxItVendorAccess,
+} from "../../command/medical/facility/verify-access";
+import { getOrganizationOrFail } from "../../command/medical/organization/get-organization";
+import { getFacilityOrFail } from "../../command/medical/facility/get-facility";
+import { createOrUpdateCWOrganization } from "../../external/commonwell/command/create-or-update-cw-organization";
+import { get as getCWOgranization, parseCWEntry } from "../../external/commonwell/organization";
+import { cwOrgActiveSchema } from "../../external/commonwell/shared";
+import { requestLogger } from "../helpers/request-logger";
+import { asyncHandler, getFrom } from "../util";
+import { getUUIDFrom } from "../schemas/uuid";
+
+const router = Router();
+
+/**
+ * GET /internal/commonwell/organization/:oid
+ *
+ * Retrieves the organization with the specified OID from CommonWell.
+ * @param req.params.oid The OID of the organization to retrieve.
+ * @returns Returns the organization with the specified OID.
+ */
+router.get(
+  "/organization/:oid",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const oid = getFrom("params").orFail("oid", req);
+
+    const org = await getOrganizationOrFail({ cxId });
```

Comment:
Could we create function to get the org by oid as well to not have to do `org.oid !== oid`

Reply:
Done.